### DISCUSSION TITLE
Nerd Fonts for Command Prompt

### DISCUSSION THREAD

**Discussion by DBragz** [JanDeDobbeleer/oh-my-posh#3063] [Troubleshoot]
Hi Everyone!

Just doing a little tinkering on my end over here with oh-my-posh and I ran into a little hurdle. I was able to load in oh-my-posh into the Windows Command Prompt application but it provides me with the [?] icons which means that oh-my-posh is not loading the Nerd Fonts I have on my machine. If I load the Command Prompt shell inside VSCode it does seem to work which is great! However, I was wondering if this functionality exists utilizing just the cmd application? Thanks!

-Dan

**Comment by JanDeDobbeleer** [comment]
@DBragz oh-my-posh can't load that for you, you need to configure your terminal to use the font you like.

**Reply by DBragz** [reply]
Understood! Yeah I changed the fonts manually in properties and now it works like a charm! Thank you @JanDeDobbeleer the program is pretty epic!
